### PR TITLE
Interactive bots: Improve task description

### PR DESCRIPTION
**Pull request by fazerlicourice7**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@fazerlicourice7, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by isht3** [comment]
Thank you for doing this! Task approved.

File: tasks/interactive-bots.md [hunk 1]
```diff
@@ -103,7 +104,7 @@ of:
  - messages sent by the bot in the `followup` stream
  - any other screenshots you find relevant
 
-Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
+* Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: I don't think we need `*` here - it was intentional to not have it.

File: tasks/interactive-bots.md [hunk 2]
```diff
@@ -39,18 +39,19 @@ There are four types of tasks in this category, each corresponding to one of the
 Follow the tutorial below to test run the followup bot on your Zulip instance
 (either local or in your droplet).
 
-* Run the Zulip server and log in the browser as Cordelia.
+* Run the Zulip server and log in to the browser as Cordelia.
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: I don't think `to` is needed here.

**Comment by trueskawka** [comment]
Went ahead and merged it with small changes.

Thanks, @fazerlicourice7!

**Pull request closed without merging**